replicatedhq/ship
# fix section headers

What I Did
------------

fix section headers in generated documentation to reflect ship-www layout

How I Did it
------------

update templates and markdown builders in hack/docs/

How to verify it
------------

    cd hack/docs && make pipeline copy-help-center-gopath

make sure the links in reference docs headers work

Description for the Changelog
------------

fix section headers in generated documentation to reflect ship-www layout

:boat:
